packages/teams-js/src/public/teamsAPIs.ts

```diff
@@ -46,12 +46,14 @@ export namespace teamsCore {
   }
 
   /**
-   * @hidden
    * Registers a handler to be called when the page has been requested to load.
    *
+   * @see Check out [App Caching in Teams](https://learn.microsoft.com/en-us/microsoftteams/platform/apps-in-teams-meetings/build-tabs-for-meeting?tabs=desktop%2Cmeeting-chat-view-desktop%2Cmeeting-stage-view-desktop%2Cchannel-meeting-desktop#app-caching)
```

Comment:
>/en-us [](http://example.com/codeflow?start=70&length=6)

Please remove the locale portion of this URL. The docs site will automatically redirect the user to the locale page most appropriate for them. Thank you! :)